tests/dynamic_data/publications/timeseries/timeseries_test.py

```diff
@@ -82,7 +82,7 @@
         'expected_thumbnail': os.path.join(DIRECTORY, 'thumbnail_timeseries.png'),
         'wms_bbox': [1743913.19942603237, 6499107.284021802247, 1755465.937341974815, 6503948.597792930901, ],
     },
-    'longname_one_file': {
+    'longname_one_file_compressed': {
```

Comment:
Put sample_png_aux_rgba_opaque.png + *.aux.xml in ZIP file instead of current TIFF

Reply:
or sample_jpg_aux_rgba.jpg